src/runtime/manifest.md

```diff
@@ -44,48 +44,47 @@ that it provides and consumes and the location of the particle implementation.
 
 ```
 particle MyParticle in 'my-particle.js'
-  in MyThing myinthing
-  out [MyThing] myoutthing
-  inout [AnotherThing] anotherthing
+  myinthing: reads MyThing
+  myoutthings: writes [MyThing]
+  otherthings: reads writes [AnotherThing] 
 ```
 
 ### Slots
-Particles that produce UI must define which slots they use for rendering. Slots may be declared required or optional. Even if all slots are optional, at least one of them must be provided in order for the particle to be instantiated. Each slot may be consumed by multiple particles.
+Particles that produce UI must define which slots they use for rendering. Slots may be declared required (default) or optional. Even if all slots are optional, at least one of them must be provided in order for the particle to be instantiated. Each slot may be consumed by multiple particles.
 
 "root" slot is a special slot that is provided by the system before any of the particles are rendered. If particle renders and creates slots for other particles to use, they are also defined in the manifest. The provided slots may restrict the handles to be rendered in the slot. The particle that consumes the provided slot must have the same handle bounded as one of its connections.
 
 ```
 particle MyParticle in 'my-particle.js'
-  in MyThing myinthing
-  out [MyThing] myoutthing
-  must consume mySlot
-    provide innerSlot
-    provide restrictedInnerSlot
-      handle myinthing
-  consume otherSlot
+  myinthing: reads MyThing
+  myoutthings: writes [MyThing]
+  mySlot: consumes  // required
+    innerSlot: provides?
+    restrictedInnerSlot: provides? Slot {handle: myinthing}
+  otherSlot: consumes?  // optional
 ```
 
-"Set slot" are a special type of slot that is provided for a handle that contains a collection of entities. A separate slot will be created for each individual element in the handle. The consuming particle must be explicitly defined to consume a set slot as well.
+A collection of slots will be provided for a handle that contains a collection of entities. A separate slot will be created for each individual element in the handle. The consuming particle must be explicitly defined to consume a colleciont of slots as well.
 
 ```
 particle MySetParticle in 'my-set-particle.js'
-  in [MyThing] mything
-  consume mySlot
-    provide set of innerSlot
+  mything: reads [MyThing]
+  mySlot: consumes
```

Comment:
Shoudl we say `consumes Slot` here, even though it is implied?

Reply:
done (yes, i agree that it is more readable)